bql/grammar/grammar_test.go

```diff
@@ -319,14 +354,30 @@ func TestAcceptGraphOpsByParseAndSemantic(t *testing.T) {
 			 ?s "old_predicate_2"@[,] ?o2.
 			 ?s "old_predicate_3"@[,] ?o3};`, empty, []string{"?b"}, []string{"?a"}, 0},
```

Comment:
Done. The current indentation is after gofmt. Wonder if it should be indented more to the left same col as the statement param?